File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.MessagingService.Models/get-notification-app-role-context_result.ts


import { NotificationAppRoleContext_Item } from './notification-app-role-context_item';
import { IGetNotificationAppRoleContext_Result } from './iget-notification-app-role-context_result';

export class GetNotificationAppRoleContext_Result implements IGetNotificationAppRoleContext_Result {
  
	public NotificationID: number = 0;
	public ContextID: number = 0;
	public NotificationContextType: string = "";
	public Item?: NotificationAppRoleContext_Item;
  
}

File: GTTS/INL.MessagingService/Models/GetNotificationAppRoleContext_Result.cs

namespace INL.MessagingService.Models
{
    public class GetNotificationAppRoleContext_Result : IGetNotificationAppRoleContext_Result
    {
        public long NotificationID { get; set; }
        public long ContextID { get; set; }
        public string NotificationContextType { get; set; }
        public NotificationAppRoleContext_Item Item { get; set; }
    }
}
